addons/mail/static/src/js/m2x_avatar_user.js

```diff
@@ -191,15 +191,6 @@ export const Many2OneAvatarUser = Many2OneAvatar.extend(M2XAvatarMixin, {
     }
 });
 
-export const KanbanMany2OneAvatarUser = Many2OneAvatarUser.extend({
-    _template: 'mail.KanbanMany2OneAvatarUser',
-
-    init() {
-        this._super(...arguments);
-        this.displayAvatarName = this.nodeOptions.display_avatar_name || false;
```

Comment:
aren't you losing some behavior in legacy ?

Reply:
Indeed, I added it back good catch